terraform/main.tf

```diff
@@ -491,8 +501,9 @@ module "arpa_exporter" {
     access_point_id = module.api.efs_data_volume_access_point_id
   }]
   additional_task_role_json_policies = {
-    rw-audit-reports-bucket = data.aws_iam_policy_document.arpa_audit_report_rw_reports_bucket.json
-    send-emails             = module.api.send_emails_policy_json
+    list-audit-reports-bucket = data.aws_iam_policy_document.arpa_audit_report_list_bucket.json
```

Comment:
Needs to be added to arpa_exporter module instead

Reply:
@as1729 It actually is (caught me off guard too for a minute) – I think there's something weird in the diff view.